client/js/components/pages/HomePage.js

```diff
@@ -1,348 +1,32 @@
 import React, { Component } from 'react';
+import getGoogleApiClient from 'google-client-api';
 
-class HomePage extends Component {
-  constructor() {
-    super();
-    this.state = { visibility: false };
-  }
+import Logo from '../../../img/xteam-vertical.png';
+import 'file?name=[name].[ext]!../../../img/xteam-vertical.png';
 
-  hidePanel() {
-    this.setState({ visibility: true });
+class HomePage extends Component {
+  componentDidMount() {
+    getGoogleApiClient(gapi => {
+      gapi.load('signin2', () => {
+        gapi.signin2.render('g-signin2', {
+          longtitle: true,
+          width: 250,
+          height: 50
+        });
+      });
+    });
   }
 
   render() {
-    const visibility = this.state.visibility ? 'hidden' : '';
     return (
```

Comment:
I don't like it. Now we have 2 components that look almost the same:
- HomePage, which is a whole routed page
- Loading, which is just a static component
We need to restructure that. I'll think how that can be done.